src/modules/tos/services/tos.service.spec.ts

```diff
@@ -32,28 +31,45 @@ describe('TosService', () => {
 
 	describe('adapt', () => {
 		it('can adapt a hasura response to our tos interface', () => {
-			const adapted = tosService.adapt(tos);
+			const adapted = tosService.adapt('1997-01-01T00:00:00.000Z');
```

Comment:
save this date as a mock constant at the beginning of the file

Reply:
fixed